tests/NPSP.py

```diff
@@ -867,4 +867,24 @@ def click_element_with_locator(self, path, *args, **kwargs):
         """Pass the locator and its values for the element you want to click """
         locator=self.get_npsp_locator(path, *args, **kwargs)  
         self.selenium.click_element(locator)      
-            
\ No newline at end of file
+            
+    def wait_for_record_to_update(self, id, value):
+        """Waits for specified record header to be updated by checking every second for 10 times.
+        """
+        el = None
+        i = 0
+        while el is None:
+            i += 1
+            if i > 10:
+                raise AssertionError(
+                    "Timed out waiting for account name to be {} .".format(value)
+                )
+            self.salesforce.go_to_record_home(id)
+            try:
+                self.verify_header(value)
+                break
+            except :
+                time.sleep(1)
+                continue     
```

Comment:
There's no need for the `continue` statement. Since this is the last line of the `while` loop, it will continue anyway!